### PR TITLE
fix(Webhook): Unit tests

### PR DESCRIPTION
**Pull request by lukas-becker0**
broken because we added the missing errors property in the last PR

### CODE REVIEW

File: __tests__/utils.ts [hunk 1]
```diff
@@ -55,6 +59,7 @@ export const webhookBodyFields: {
 		},
 	},
 	status: {
+		errors: [webhookError, webhookError],
```
Comment (MarcosNicolau) [inline __tests__/utils.ts:62]: Why is the err added twice?

Comment (lukas-becker0) [reply __tests__/utils.ts:62]: Only because the errors property under `changes[0].value.errors` also contains the error twice.

Comment (lukas-becker0) [reply __tests__/utils.ts]: I changed it to one entry

**Comment by MarcosNicolau** [comment]
Great, thank you!

**Pull request merged**